src/test/java/com/example/enterprise/MoneyManagerApplicationTests.java

```diff
@@ -54,18 +50,19 @@ void addAndSaveANewIncome(){
 
         incomeService.save(income);
 
-        List<Income> incomeEntries =incomeService.listAllIncomes();
+        List<Income> incomeEntries = incomeService.listAllIncomes();
         boolean checkNewIncome = false;
-        for(Income i : incomeEntries){
-            if(i.getSource().equals(incomeSource) && i.getIncomeID() == id){
+        for (Income i : incomeEntries) {
+            if (i.getSource().equals(incomeSource) && i.getIncomeID() == id) {
```

Comment:
Fixing the minor whitespaces for clean coding conventions and practices. 

Whitespaces practices can be found in this link: https://www.baeldung.com/java-clean-code